src/examples/table/TableScroll.tsx

```diff
@@ -24,22 +24,22 @@ const rowData: IRow[] = Array.from(Array(100)).map((row, index) => ({
 const Example = () => (
   <div style={{ maxHeight: 500, overflow: 'auto' }}>
```

Comment:
This needs `color-scheme` that follows light/dark. In `react-components`, we used `only light` and `only dark`, respectively. :)